variables.tf

```diff
@@ -87,3 +87,9 @@ variable "block_everything_capacity" {
   type        = number
   default     = 25
 }
+
+variable "delete_protection" {
+  description = "Toggle to enable or disable deletion protection"
+  type        = bool 
+  default     = false 
```

Comment:
Security Hub mentions to enable deletion protection
https://docs.aws.amazon.com/securityhub/latest/userguide/networkfirewall-controls.html#networkfirewall-9
So maybe the default value should be true. And consumers of the module should make a conscious decision to disable it

Add this var to the README too with the link to the AWS doc i provided above